Configuration/PyReleaseValidation/python/relval_upgrade.py

```diff
@@ -25,8 +25,10 @@
         for frag in upgradeFragments:
             k=frag[:-4]+'_'+key
             stepList=[]
-            for step in upgradeProperties[year][key]['ScenToRun']:
+            for step in upgradeProperties[year][key]['ScenToRun']:                    
                 if 'Sim' in step:
+                    if 'HLBeamSpotFull' in step and '14TeV' in frag:
```

Comment:
It seems the only way to do it since all combinations of upgrade workflows are made and then everything is sorted out in the end. Especially since we have 14 TeV wfs mixed with 13 TeV wfs. Don't see a nicer way to do it.